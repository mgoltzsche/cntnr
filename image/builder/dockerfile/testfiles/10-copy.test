FROM alpine:3.7
COPY entrypoint.sh /entrypoint.sh
COPY --chown=root:root entrypoint.sh *.conf /
COPY --from=alpine:3.7 etc/hosts /
COPY --from=alpine:3.7 --chown=root:root /etc/passwd /

# ASSERT RUN [ -f "/entrypoint.sh" ] && [ -f "/cfg-a.conf" ] && [ -f "/hosts" ] && [ -f "/passwd" ]

# TODO: test different users (need to run test as root or use proot)