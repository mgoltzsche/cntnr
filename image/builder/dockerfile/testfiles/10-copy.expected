FROM alpine:3.7
COPY dir="./ctx" "entrypoint.sh" "/entrypoint.sh" 0:0
COPY dir="./ctx" "entrypoint.sh" "*.conf" "/" root:root
COPY image="alpine:3.7" "etc/hosts" "/" 0:0
COPY image="alpine:3.7" "/etc/passwd" "/" root:root